ios/FluentUI/Vnext/Drawer/Drawer.swift

```diff
@@ -17,16 +17,12 @@ import SwiftUI
 /// `MSFDrawerState` assist to configure drawer functional properties via UIKit components.
 @objc public class MSFDrawerState: NSObject, ObservableObject {
 
-    /// A callback executed when the drawer is expanded/collapsed
-    public var onStateChange: (() -> Void)?
+    /// A callback executed after the drawer is expanded/collapsed
+    public var didChangeState: ((Bool?) -> Void)?
```

Comment:
I think onStateChange is aligned with the naming standard, isn't it?

Reply:
That's a good point. I wanted to convey that state changes are called after animation is completed as opposed to before. I like the idea of consistency with SwiftUI nomenclature.